src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorDataflow.java

```diff
@@ -118,7 +121,7 @@ protected void setupPipeline(Pipeline pipeline) {
             BaseRecalibratorDataflowUtils.ensureReferenceIsReadable(pipeline.getOptions(), referencePath);
```

Comment:
Fair enough. And that code's about to be changed anyways as it's integrated into the skeleton. I created issue #738 to track this.